storage/acl.go

```diff
@@ -351,6 +381,14 @@ func (r ACLRule) toProtoObjectAccessControl(bucket string) *storagepb.ObjectAcce
 	}
 }
 
+func (r ACLRule) toProtoBucketAccessControl() *storagepb.BucketAccessControl {
+	return &storagepb.BucketAccessControl{
+		Entity: string(r.Entity),
```

Comment:
Is this because these are enums currently and will need to change after the breaking change?

Reply:
No, it's because `r.Entity` is an `ACLEntity` - a type alias for `string`. The proto field is a `string` and the compiler won't accept an `ACLEntity` so we must coerce it to a `string`

Reply:
Ahh of course, sorry I wasn't tracking this very closely haha.